### ISSUE TITLE
The element path should have an appropriate aria-label to support assistive devices.

### ISSUE THREAD

**Issue by scott-redrup-ou** [tinymce/tinymce#10105]
## 📝 Provide a description of the new feature or improvement
The element path functionality at the bottom of the TinyMCE editor does not provide any context about its purpose to screen reader users.  

When navigating to the element path using a screen reader a description of the feature should be read aloud to the user. 

### 🫶 What is the motivation?
To support screen reader users in making use of the full functionality in TinyMCE.  

### 🔗 What is the consequence of not having this feature?
The text read aloud when screen reader users navigate to this area currently is unhelpful and confusing e.g. ‘p button’, ‘ul button’. 

### 🚦 How important would you rate the requested feature or improvement?
Important
